libraries/textcomposer/impl/src/main/kotlin/io/element/android/libraries/textcomposer/TextComposer.kt

```diff
@@ -146,7 +146,7 @@ fun TextComposer(
             val roundCornerLarge = 28.dp.applyScaleUp()
 
             val roundedCornerSize = remember(state.lineCount, composerMode) {
-                if (state.lineCount > 1 || composerMode is MessageComposerMode.Special) {
```

Comment:
Are we sure we want to have large rounded corners for multi-line composer state?

<img width="517" alt="image" src="https://github.com/vector-im/element-x-android/assets/480955/abef6012-f14d-4619-ae08-9fa516d91f37">

vs

<img width="519" alt="image" src="https://github.com/vector-im/element-x-android/assets/480955/0d7049db-4861-4165-bd4a-90788cc11aae">

Reply:
Yeah the problem we tried to fix with this is that the shape of the editor changes when switching between 1 and 2 lines which was a bit jarring

Reply:
Ok, it sounds a bit weird to me to make the transition animation more important than the UI the user will see all the time while having several lines, but if the design team asked for this change then 🤷 .